### ISSUE TITLE
Question: format of SHFB Website output

### ISSUE THREAD

**Issue by kevin-galil** [EWSoftware/SHFB#64]
Hi,
This is really a question on website type output, not an issue with the software.
I did a quick scan of HTML output and I think it contains java scripts with normal HTML formats.  The output type stays ASP.NET and when I run index.html, IE asks me if I want to enable ActiveX object.  And it looks like it's related to moving the divider that splits the screen between right and left.

I'm wondering if the web output require anything on server's end to run.  Our web server is Apache on LINUX and the policy is that anything runs on server-side is a no-no for us.

Thank you for your help.

**Comment by EWSoftware** [comment]
Locally, when choosing to view the content after the build, you can choose to run it by just opening Index.html on the file system or by starting the web development server and opening it.  Browsers will block a lot of stuff when opening content from the local file system and will prompt you to enable ActiveX support since it does make use of it in a few locations in the script for backward compatibility in the asynchronous calls.  That's why there's an option to run it using the web development server which runs without the extra local file system restrictions.  For the VS2010 and earlier styles, there is an Index.aspx page which is given precedence when using the development server.  The VS2013 style only contains an Index.html page so it gets used either way.

Once you upload it to an actual web server, you can open Index.html, index.aspx, or index.php (VS2010 or earlier styles).  If you use Index.html, you can still view the site but it lacks the search and index options.  For the VS2013 style, there's only an Index.html page that redirects to the default topic and it doesn't use a frameset so there's no ASP.NET or PHP page hosting the content.

For all presentation styles, there are ASP.NET and PHP server side search pages including the VS2013 style.  However, for the VS2013 style, if the client-side script can't access either of those, it falls back to client-side script to download the index files and perform the search without them.  Slower but it still works.  There's no index page option in the VS2013 style.

Bottom line, nothing special is required on the server to support the output if you choose not to enable ASP.NET or PHP support.  You can delete the .aspx and .php pages from the build output before or after uploading the content and it'll still work.  The VS2013 style is the best choice in this case since it still contains a working search option.

**Comment by kevin-galil** [comment]
Thank you for the detailed answer and your help.  VS2013 style is the one we are interested in :)
